pkg/issuer/venafi/client/venaficlient.go

```diff
@@ -217,16 +252,19 @@ func configForIssuer(iss cmapi.GenericIssuer, secretsLister internalinformers.Se
 //     because cert-manager establishes a new HTTPS connection for each API
 //     request and therefore should only ever need to renegotiate once in this
 //     scenario.
-//  5. But overriding the HTTP client causes vcert to ignore the
+//
+// Why do we supply CA bundle in the HTTP client **and** in the vcert.Config?
```

Comment:
Do we still have to pass the cabundle through vcert.Config?
If we unconditionally pass a http client with the cabundle configured, I think we don't have to pass the CAs through vcert.Config anymore.

Reply:
EDIT: this comment explains why we pass it twice: https://github.com/cert-manager/cert-manager/pull/6865/files#diff-5f8cbfd3b75b6d8e0f899a92377a3aa5b5fa1867e18d19fcda51f8e8db2793b0L149-L156
Maybe we can refer to that comment here or add some extra info?